### PR TITLE
Week13/list race condition

### PR DESCRIPTION
**Pull request by LianaSimonian**

### CODE REVIEW

File: untitled/main/com/aca/homework/week13/race/condition/list/List.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.aca.homework.week13.race.condition.list;
+
+public interface List<E> {
+
+    void add(E item);
+
+    int size();
+
+    E get(int index);
```
Comment (gevorgyanarman) [inline untitled/main/com/aca/homework/week13/race/condition/list/List.java:9]: this method was not in requirement

File: untitled/main/com/aca/homework/week13/race/condition/list/List.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+    boolean contains(E item);
```
Comment (gevorgyanarman) [inline untitled/main/com/aca/homework/week13/race/condition/list/List.java:11]: this method was not in requirement

File: untitled/main/com/aca/homework/week13/race/condition/list/NumberAdderRunnable.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package com.aca.homework.week13.race.condition.list;
+
+public class NumberAdderRunnable implements Runnable {
+
+    List<Integer> list;
```
Comment (gevorgyanarman) [inline untitled/main/com/aca/homework/week13/race/condition/list/NumberAdderRunnable.java]: should be private

**Review by gevorgyanarman — CHANGES_REQUESTED** [review]
please add private keyword only and merge the pull request

File: untitled/main/com/aca/homework/week13/race/condition/list/List.java (hunk 1 above)
Comment (LianaSimonian) [reply untitled/main/com/aca/homework/week13/race/condition/list/List.java:9]: I had to add it so I could write a @Test

Comment (LianaSimonian) [reply untitled/main/com/aca/homework/week13/race/condition/list/List.java:9]: I had to add it to be able to write a correct @Test

File: untitled/main/com/aca/homework/week13/race/condition/list/List.java (hunk 2 above)
Comment (LianaSimonian) [reply untitled/main/com/aca/homework/week13/race/condition/list/List.java:11]: I had to add it to be able to write a correct @Test

**Pull request merged**